### PR TITLE
Create "Log2Lin" node

### PR DESCRIPTION
**Pull request by mrjschulte**
This node will apply the Kodak Cineon set of Logarithmic functions to an image.

### CODE REVIEW

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+from nodes.properties.inputs import BoolInput, ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:log2lin",
+    description="Convert all channels in an image to a scene linear encoding using the Cineon logarthmic function set.",
```
Comment (RunDevelopment) [inline backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py:13]: Typo and could you explain this a bit more? I have no idea what this is for from just reading the description. (Descriptions are markdown, so you can also link to websites to do the explaining for you.)

```suggestion
    description="Convert all channels in an image to a scene linear encoding using the Cineon logarithmic function set.",
```

**Comment by mrjschulte** [comment]
The mentioned Kodak Cineon functions are a means to help convert the logarithmic densities encoded on a 0-1 real world medium such as Film Negative into a linear floating point encoding for use in imaging workflows , and vice versa.

**Comment by RunDevelopment** [comment]
Then could you please mention this in the node's description?

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+from nodes.properties.inputs import BoolInput, ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import adjustments_group
+
+
+@adjustments_group.register(
+    schema_id="chainner:image:log2lin",
+    description="Convert all channels in an image to a scene linear encoding using the Cineon logarithmic function set.",
+    name="Log2Lin",
```
Comment (RunDevelopment) [inline backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py:14]: I think we should call this node "Log To Linear" to be consistent with names of other nodes (e.g. "Metal To Specular", "ONNX To NCNN", "Text As Image").

Comment (mrjschulte) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py:14]: Elaborating on this note, I have a more general question about the naming schema for nodes: I noticed a lot of the nodes are also using additional verbs such as "Change", "Create", "Add", etc. Is there some sort of specific thinking around this approach so far? Or is it solely based on providing clarity to the Nodes function?

Comment (joeyballentine) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py]: We don't have any kind of consistent system right now, we just do whatever seems right at the time. We've never really gone back and reviewed anything or tried to make it consistent

**Comment by joeyballentine** [comment]
Just FYI, we require all node names to match their function name and file name for ease of searchability. This will actually automatically be changed for you if you run with `npm run dev` after/while you update it

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py (hunk 2 above)
Comment (mrjschulte) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/log2lin.py]: Node renamed and updated explanations and other items. Relinted. Gtg from my pov.

**Review by joeyballentine — APPROVED** [review]

**Pull request merged**